sources/org.osbuild.skopeo

```diff
@@ -64,18 +64,22 @@ SCHEMA = """
 """
 
 
-def download(items, cache):
-    for image_id, item in items.items():
-        image = item["image"]
+class SkopeoSource(sources.SourceService):
+
+    content_type = "org.osbuild.containers"
+
+    # pylint: disable=arguments-renamed
+    def fetch_one(self, image_id, desc):
+        image = desc["image"]
         imagename = image["name"]
         digest = image["digest"]
         tls_verify = image.get("tls-verify", True)
 
-        path = f"{cache}/{image_id}"
+        path = f"{self.cache}/{image_id}"
         if os.path.isdir(path):
-            continue
+            return
```

Comment:
Is this still needed? We should have filtered existing ones, no?

Reply:
indeed!